samples/CoreAnimationTest/CoreAnimationTest/LayerTableViewCell.m

```diff
@@ -26,7 +26,7 @@ @implementation LayerTableViewCell {
 
 - (instancetype)initWithStyle:(UITableViewCellStyle)style reuseIdentifier:(NSString*)reuseIdentifier {
     if (self = [super initWithStyle:style reuseIdentifier:reuseIdentifier]) {
-        self.backgroundColor = [UIColor whiteColor];
+        self.backgroundColor = [UIColor clearColor];
```

Comment:
This was suggested by Nick - let me see if I can spot whether removing it has an impact on iOS or IW.

---
In reply to: [99437839](https://github.com/Microsoft/WinObjC/pull/1874#discussion_r99437839) [](ancestors = 99437839)